src/components/DpDateRangePicker/DpDateRangePicker.vue

```diff
@@ -53,12 +53,9 @@ export default {
     },
 
     dataCy: {
-      type: Object,
+      type: String,
       required: false,
-      default: () => ({}),
-      validator: (prop) => {
-        return Object.keys(prop).every(key => ['endDate', 'startDate'].includes(key))
-      }
+      default: 'dataRangePicker'
```

Comment:
I like the idea of using the component name as default!

```suggestion
      default: 'dateRangePicker'
```